From 57e7cb9f1909daa5798b3c26b52c07ee0e26591e Mon Sep 17 00:00:00 2001
From: Abhishek Kondaveeti <akondave@codeaurora.org>
Date: Sat, 21 Dec 2013 22:07:08 +0530
Subject: [PATCH 078/176] mm-camera-test: Eztune: Fixing preview resolution.

We need to use the preview resolution for the preview,
and we shouln't take from buffer_width as this variable
only valid for snapshot.

Change-Id: Ief8d49a93aafbd429112484be89c79eefc34329b
---
 QCamera2/stack/mm-camera-test/src/mm_qcamera_socket.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/QCamera2/stack/mm-camera-test/src/mm_qcamera_socket.c b/QCamera2/stack/mm-camera-test/src/mm_qcamera_socket.c
index ae5c729..6bbf748 100644
--- a/QCamera2/stack/mm-camera-test/src/mm_qcamera_socket.c
+++ b/QCamera2/stack/mm-camera-test/src/mm_qcamera_socket.c
@@ -775,8 +775,10 @@ void *eztune_proc(void *data)
         }
       }*/
       cam_dimension_t dim;
-      dim.width = lib_handle->test_obj.buffer_width;
-      dim.height = lib_handle->test_obj.buffer_height;
+      //dim.width = lib_handle->test_obj.buffer_width;
+      //dim.height = lib_handle->test_obj.buffer_height;
+      dim.width = DEFAULT_PREVIEW_WIDTH;
+      dim.height = DEFAULT_PREVIEW_HEIGHT;
 
       CDBG("preview dimension info: w(%d), h(%d)\n", dim.width, dim.height);
       // we have to make sure that camera is running, before init connection,
-- 
1.8.3.1

